comms/lrzsz/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                lrzsz
 version             0.12.20
-revision            4
+revision            5
```

Comment:
Since you're fixing an issue with a specific compiler, you don't need to increase the revision.